lib/scrabble.rb

```diff
@@ -1,9 +1,24 @@
 class Scrabble
 
-  def score(word)
-    1
+  def score(word = nil)
+    if word == nil || word == ""
+      return 0
+    else
+      letters = word.upcase.split('')
+      value = letters.map { |letter| point_values.values_at(letter)[0] }
```

Comment:
You'll more commonly see point_values[letter] instead of point_values.values_at(letter), it's a little more straight forward.